src/App.jsx

```diff
@@ -3,26 +3,72 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { request } from './components/API/api';
+
+const URL = 'https://mate-api.herokuapp.com';
```

Comment:
Keep URL and endpoint settings in api file, here you just need to import necessary functions.